pkg/backup/item_backupper.go

```diff
@@ -484,10 +484,9 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 	}
 
 	if ib.backupRequest.ResPolicies != nil {
-		structuredVolume := &resourcepolicies.StructuredVolume{}
-		structuredVolume.ParsePV(pv)
-		action := ib.backupRequest.ResPolicies.Match(structuredVolume)
-		if action != nil && action.Type == resourcepolicies.Skip {
+		if action, err := ib.backupRequest.ResPolicies.GetMatchAction(pv, resourcepolicies.PersistentVolumes); err != nil {
+			log.WithError(err).Errorf("Error getting matched resource policies for pv %s", pv.Name)
```

Comment:
Should we return directly or just log the error?

Reply:
yes, I've changed to  return directly